src/Illuminate/Support/Number.php

```diff
@@ -367,6 +367,30 @@ public static function defaultCurrency()
         return static::$currency;
     }
 
+    /**
+     * Generate a random number of the given length.
```

Comment:
"number" might be misleading here:
```suggestion
     * Generate a random integer of the given length.
```